3/collection-meta.yaml

```diff
@@ -1,5 +1,8 @@
 ---
 collections:
+  dellemc.openmanage:
+    maintainers:
```

Comment:
```suggestion
    changelog-url: https://github.com/dell/dellemc-openmanage-ansible-modules/blob/collections/CHANGELOG.rst
    maintainers:
```

Reply:
@felixfontein I might have assumed that this wasn't required for collections that are using antsibull-changelog for relnotes given that the other changelogs in the file are markdown. Should we add it anyway ?

Reply:
They do not use antsibull-changelog, since they do not have a changelogs/changelog.yaml file.

Reply:
OH, you're right, I don't know how I missed that.

Reply:
The CHANGELOG.rst file really looks like is created by antsibull-changelog :)